mir_eval/separation.py

```diff
@@ -328,7 +438,8 @@ def _safe_db(num, den):
     return 10 * np.log10(num / den)
 
 
-def evaluate(reference_sources, estimated_sources, **kwargs):
+def evaluate(reference_sources, estimated_sources,
+             framewise=False, window=30*44100, hop=15*44100, **kwargs):
```

Comment:
> For 2. is it better if I return a nested dictionary or should each part have it's own name?

It should return a flat dictionary, each metric having its own name, e.g. `'Source to Distortion (framewise)'`, etc.

> (Also, this approach will not work for images support, as the existing function cannot be called on images...)

Why is that?  We can discuss this elsewhere (probably in the images issue).